src/app/@core/models/CreatedActivity.model.ts

```diff
@@ -0,0 +1,21 @@
+export class CreatedActivity {
```

Comment:
rename this file to createActivity.model.ts

Reply:
changed file name